test/0.4.24/helpers/publicKeyArrays.js

```diff
@@ -0,0 +1,14 @@
+const { hexConcat, padKey, padSig } = require('../../helpers/utils')
+
+const createKeys = (numKeys) => Array.from({ length: numKeys }, (_value, i) => padKey(`0x${i + 1}`))
+const createSigs = (numSigs) => Array.from({ length: numSigs }, (_value, i) => padSig(`0x${i + 1}`))
+
+const sanitiseKeyArray = (keys) => hexConcat(...keys.map((key) => padKey(key)))
```

Comment:
I think `sanitiseKeyArray` is not a good name for this fn, I'd call it like `packKeys` and `packSigs`.

Reply:
Also, I'd not extract these utilities into a separate file: this makes it harder to digest what actually happens in tests.

Reply:
Good point on the name. I was focusing on the padding but this function does more than that.

This does need to be pushed out to a utils file as when dealing with offchain keys we're going to be manipulating much longer arrays of keys than current so this function would otherwise be duplicated.